src/assets/data/workshops-info.js

```diff
@@ -1,4 +1,15 @@
 let workshopsData = [
+    {
+        title: "Tech for Good Panel", 
+        description: "Come join us for our Tech for Good panel with inspiring women from Change.org, LaunchCode, and Nina Space! We will be exploring the side of tech that brings good changes into our everyday lives.",
```

Comment:
```suggestion
        description: "Come join us for our Tech for Good panel with inspiring women from Change.org, LaunchCode, and Cosmos Pics We will be exploring the side of tech that brings good changes into our everyday lives.",
```
Should be Cosmos Pics right?

Reply:
fixed